src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -129,6 +129,16 @@ const PersonalDetails = (() => {
         }
     };
 
+    const modifyContactCSMessage = (immutable_fields) => {
+        if (!/tax/.test(immutable_fields.toString())) {
+            $('.rowCustomerSupport').text((index, text) => text.replace(localize('To change your name, date of birth, country of residence, email, or tax information'),
+                localize('To change your name, date of birth, country of residence, or email')));
+        }
+        if (!/name/.test(immutable_fields.toString())) {
+            $('.rowCustomerSupport').text((index, text) => text.replace(localize('To change your name, date of birth'), localize('To change your date of birth')));
```

Comment:
else if?

Reply:
I expect both 'if' to fire if both of them are truthy. 'Else if' will make the second part be skipped. That's why I used 'if's only.

Reply:
it seems that both if's are changing $('.rowCustomerSupport') element. therefore if second if is true, it will overwrite first if

Reply:
I expect it as well. that's why the sequence of 'if's is important here.